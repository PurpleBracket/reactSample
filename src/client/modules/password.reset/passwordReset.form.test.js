/* global describe test expect */
import React from 'react'
import { shallow } from 'enzyme'
import PasswordResetForm from './passwordReset.form'

describe('Register', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(<PasswordResetForm />)
    expect(wrapper).toMatchSnapshot()
  })
})
