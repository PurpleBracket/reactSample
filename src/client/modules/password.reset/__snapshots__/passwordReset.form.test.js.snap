// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register should match snapshot 1`] = `
<div>
  <div
    className="columns"
  >
    <div
      className="column"
    />
    <div
      className="column is-one-third"
      style={
        Object {
          "minHeight": "100vh",
        }
      }
    >
      <Panel
        className="passwordReset"
        noBorder={true}
        noShadow={true}
        transparentBG={true}
      >
        <Subhead
          size="4"
        >
          Reset Your Password
        </Subhead>
        <form
          className="card-content"
          onSubmit={[Function]}
        >
          <Input
            hasIconLeft={true}
            hasIconRight={true}
            icon="envelope"
            name="email"
            onChange={[Function]}
            placeholder="Email Address"
            required={true}
            type="email"
          />
          <div
            className="field buttons is-centered"
          >
            <Button
              color="white"
              style=""
              type="submit"
            >
              Reset Password
            </Button>
          </div>
        </form>
        <footer
          className="card-footer"
        >
          <p
            className="card-footer-item"
          >
            <span>
              <LinkAtom
                href="/signup"
              >
                Don't have an account? 
                <br />
                 
                <b>
                  Create an account
                </b>
              </LinkAtom>
            </span>
          </p>
        </footer>
      </Panel>
    </div>
    <div
      className="column"
    />
  </div>
</div>
`;
